src/navigation/TabNavigator.tsx

```diff
@@ -49,24 +50,39 @@ const TabNavigatorOptions: BottomTabNavigationOptions = {
 };
 
 const HomeScreenOptions: BottomTabNavigationOptions = {
-  tabBarIcon: TabBarIcon.ionicons("home-outline", "home"),
+  tabBarIcon: (props) => {
```

Comment:
Props isn't typed here. Can you please type them?

Reply:
It was typed via BottomNavigationOptions I think? I'll double check

Reply:
Oh, perfect. So long as we have autocomplete, I'm good with it!